std/container/slist.d

```diff
@@ -589,6 +602,39 @@ Complexity: $(BIGOH n)
 
 /// ditto
     alias stableLinearRemove = linearRemove;
+
+/**
+Removes an element from the list in linear time.
```

Comment:
Define behavior when the value occurs multiple times (e.g. the list is iterated from front to back and the first match is removed).

Reply:
For example:

```
Removes the first an element from the list that equals `value` in $(BIGOH n) time.
```